tasks/updateconstants.js

```diff
@@ -669,6 +676,106 @@ const sources = [
       return result;
     },
   },
+  {
+    key: "aghs_desc",
+    url: aghs_desc_urls,
+    transform: (respObj) => {
+      const herodata = respObj;
+      aghs_desc_arr = [];
+
+      // for every hero
+      for(i=0;i < herodata.length; i++)
+      {
+        // check validity of data from URL
+        // some hero IDs are unused (to the public ;) )
+        if(herodata[i].result.data.heroes.length == 0 || herodata[i].result.data.heroes[0].abilities.length == 0)
+        {
+          continue;
+        }
+        
+        hd_hero = herodata[i].result.data.heroes[0];
+        
+        // object to store data about aghs scepter/shard for a hero
+        var aghs_element = 
+        {
+          hero_name:    "",
```

Comment:
can you just set these values from hd_hero directly?

Reply:
This was just to declare the structure of the object, but I guess I could just straight up initialize it with what data I have, here. I'll see what I can do